nautobot/ipam/tests/test_api.py

```diff
@@ -343,7 +343,7 @@ def test_create_single_available_ip(self):
             }
             response = self.client.post(url, data, format="json", **self.header)
             self.assertHttpStatus(response, status.HTTP_201_CREATED)
-            self.assertEqual(str(response.data["parent"]), self.absolute_api_url(prefix))
+            self.assertEqual(str(response.data["parent"]["url"]), self.absolute_api_url(prefix))
```

Comment:
optional, but shall we check the `object_type` and the `id`  attributes are rendered correctly as well?

Reply:
Same feedback as above